ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/JsonCallParameter.java

```diff
@@ -40,6 +40,31 @@ public class JsonCallParameter extends CallParameter {
 
   private final Optional<Boolean> strict;
 
+  public JsonCallParameter(
```

Comment:
Is this used anywhere outside of tests?  Can we just rip the band-aid off and migrate to the new signaure?  IntelliJ has a tool for this Refactor>Change Signature... (command/alt F6)

Reply:
Only tests, changing class to only use a single constructor. thanks for the feedback.